applications/volume_rendering_xr/Dockerfile

```diff
@@ -27,36 +27,6 @@ COPY applications/volume_rendering_xr/thirdparty/magicleap/MagicLeapRemoteRender
     /usr/local/share/keyrings/magicleap/MagicLeapRemoteRendering.gpg
 RUN /tmp/base-setup.sh
 
-############################################################
```

Comment:
Could you remove the `ARG VULKAN_SDK_VERSION=1.3.216.0` above as well. That's no longer needed.

Reply:
Removed